package.json

```diff
@@ -1,109 +1,109 @@
 {
-  "name": "canvas-datagrid",
```

Comment:
Indentation here is off, again — can you restore the original indentation, so the diff shows properly?